src/components/OpenWith/OpenWith.js

```diff
@@ -115,12 +211,15 @@ class OpenWith extends PureComponent<Props> {
      */
     render() {
         const { language, messages: intlMessages }: Props = this.props;
+        const { integrationsData }: State = this.state;
 
         // Placeholder
         return (
-            <Internationalize language={language} messages={intlMessages}>
-                <button> Open With </button>
-            </Internationalize>
+            integrationsData && (
```

Comment:
Minor: Should this occur immediately after `return`? Would avoid this indentation.